### PR TITLE
Locally provided CRIS extract support for import ETL

### PR DESCRIPTION
**Pull request by frankhereford**
## Associated issues

_none_, please see https://austininnovation.slack.com/archives/CHZE6BC6L/p1695994302956589. I needed the functionality this morning, and since I had already hit a road bump via leaving a file on the SFTP server yesterday, I went ahead and coded up this feature to use today.

## Purpose

Add support to CRIS import script to process locally provided zip files instead of pulling them via `rsync` from the SFTP server. This is particularly helpful in local development.

## Testing

**Steps to test:**

* Place a CRIS extract zip file in the `atd-etl/cris_import/development_extracts/` folder.
* Spin up local VZDB
* Start the `cris_import` container with `docker compose run cris-import bash`
* Run the export and observe that it doesn't access the SFTP endpoint, but rather uses the local file.

---
#### Ship list
- [ ] Run migration steps in readme in /atd-vzd/ if needed
- [ ] Code reviewed
- [ ] Product manager approved

### CODE REVIEW

File: atd-etl/cris_import/cris_import.py [hunk 1]
```diff
@@ -190,12 +204,20 @@ def get_secrets():
...
+    if not zip_files:
+        return False
+
     zip_tmpdir = tempfile.mkdtemp()
-    shutil.copy(file, zip_tmpdir)
+
+    for file_to_copy in zip_files:
+        shutil.copy(file_to_copy, zip_tmpdir)
+
     return zip_tmpdir
```
Comment (frankhereford) [inline atd-etl/cris_import/cris_import.py:217]: I actually have used this technique in the past. This spiffs up an existing `specify_extract_location()` function.

File: atd-etl/cris_import/lib/sql.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
 pp = pprint.PrettyPrinter(indent=4)
 
 # This library is /not/ designed for reuse in other projects. It is designed to increase the
-# readability of the `cris_import.py` prefect flow by pulling logical groups of code out
+# readability of the `cris_import.py` ETL process by pulling logical groups of code out
```
Comment (frankhereford) [inline atd-etl/cris_import/lib/sql.py:9]: I'm piggy-backing this comment change on this PR.

**Comment by frankhereford** [comment]
@mddilley I'll put this out to the wider group if you like it. I thought I'd ask if you'd give it a peek since we were talking about it this morning.

File: atd-etl/cris_import/cris_import.py [hunk 3]
```diff
@@ -75,7 +76,19 @@ def main():
 
     SFTP_ENDPOINT_SSH_PRIVATE_KEY = secrets["sftp_endpoint_private_key"]
 
-    zip_location = download_archives()
+    local_mode = False
+    if bool(glob.glob('/app/development_extracts/*.zip')):
```
Comment (mddilley) [inline atd-etl/cris_import/cris_import.py:80]: I tested this with two different extracts that I grabbed from the sftp server, and I saw the script pick them up with no issue. ✨
Screenshot 2023-09-29 at 12 33 07 PM

**Review by mddilley — APPROVED** [review]
Works great! This could even be useful in the non-LDM branches for developing the import script. Could be a good one to scope for the future. Specifying whether to target local or prod in the on execution would be a nice feature for down the road. Curious to see what others think too! 🚀

**Review by chiaberry — APPROVED** [review]
approving based on reading the code -- this is a good change that will prevent the import having issues due to testing!

**Review by patrickm02L — APPROVED** [review]
🚢

**Pull request merged**